### DISCUSSION TITLE
Constant speed should also work with taiko

### DISCUSSION THREAD

**Discussion by Doksha** [ppy/osu#20974] [Ideas] [answered]
The constant speed mod in mania disables scroll speed changes, the same should be available in taiko.

**Comment by Doksha** [comment, marked as answer]
duplicate post, forgot to check

**Reply by nagi-desuuu** [reply]
Duplicate of what, exactly?

**Reply by Joehuu** [reply]
https://github.com/ppy/osu/discussions/17927
